src/compiler/diagnosticMessages.json

```
@@ -3663,6 +3663,14 @@
         "category": "Error",
         "code": 2854
     },
+    "A class cannot extend a primitive type like '{0}'. Classes can only extend constructable values.": {
+        "category": "Error",
+        "code": 2855
+    },
+    "A class cannot implement a primitive type like '{0}'. It can only extend other named object types.": {

```

Should this message refer to 'implement' both times? Seems like it should but I'm not sure.

```suggestion
    "A class cannot implement a primitive type like '{0}'. It can only implement other named object types.": {
```

---

You're right, that looks a typo. 'implement' would be better in this context.